.github/workflows/build-engine-container.yml

```
@@ -230,18 +228,18 @@ jobs:
       - name: <Build/Deploy> Build and Deploy multi-platform Docker image
         env:
           IMAGE_NAME: ${{ env.IMAGE_NAME }}
-          VERSION_OR_LATEST: ${{ needs.config.outputs.version_or_latest }}
+          VERSION_OR_LATEST: ${{ needs.config.outputs.version }}
           AMD64_IMAGE_PREFIX: ${{ matrix.amd64_image_prefix }}
           ARM64_IMAGE_PREFIX: ${{ matrix.arm64_image_prefix }}
           MULTI_PLATFORM_IMAGE_PREFIXES: ${{ matrix.multi_platform_image_prefixes }}
         run: |
           bash ./tools/build_and_push_multi_platform_docker_images.bash
 
   run-release-test-workflow:
-    # version が指定されている場合のみ実行する
-    if: needs.config.outputs.version_or_latest != 'latest'
+    # FIXME: "edge"タグの場合、テストは控えるようにする
+    if: true
```

テストができないのはdockerhubにpushしない場合のみで、edgeタグを付けてdockerhubにpushすることもあるので、このコメントは意図と違うかも！！

どちらかというとこう･･･？
```suggestion
    # NOTE: 一時的にデフォルトブランチでビルドしていないので常にtrueになっている
    # FIXME: Docker Hubにpushしない場合はfalseになるようにする
    if: true
```

コミッターがこのワークフローを回すこともあるはずで、その時はpushしないのでこうなるかなと！！